src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,98 @@
+import styled, {css} from "styled-components";
+
+
+
+const theme = {
```

Comment:
Docelowo w projekcie mamy 1 motyw. Ewentualnie można definiować wartości domyślne tutaj.
Pamiętaj, że motyw to ma być rozwiązanie, które pozwoli Ci nadać kompletnie nowe "kolory i wielkości" bez potrzeby modyfikowania plików "standardowych"